### ISSUE TITLE
Load charts one after the other.

### ISSUE THREAD

**Issue by mazhar-kv** [amcharts/amcharts4#4254]
**Stack**

- React Typescript
- AmCharts4

[**CodeSandbox**
](https://codesandbox.io/s/beautiful-pine-zs1re0?file=/src/pages/Bar.tsx)

I have created multiple Xychart from Json config. Each chart is loaded based on the order they receive their data. But i want the chart to start rendering only after the previous chart is painted completely on the dom. 

I have added queue option, also added a deferred creation of the chart. But next chart begins to render before the previous one is full painted.

**Comment by martynasma** [comment]
Using [deferred](https://www.amcharts.com/docs/v4/concepts/performance/#Creating_a_deferred_chart_instance) instances should start rendering new instance only after the previous one is fully initialized.

Please note that "initialized" does not mean initial animations have fully played out, if that's what you mean.

You can add a [time gap](https://www.amcharts.com/docs/v4/concepts/performance/#Specifying_time_gap_between_deferred_chart_instantiations) between deferred creates to compensate for that.
